src/literature/containers/DetailPage/DetailPage.jsx

```diff
@@ -148,35 +155,41 @@ class DetailPage extends Component {
             </Row>
           </ContentBox>
         </Col>
-        {(numberOfReferences > 0 || citationCount > 0) && (
-          <Col className="mt3 mb3" span={14}>
-            <Tabs
-              type="card"
-              tabBarStyle={{ marginBottom: 0 }}
-              className="remove-top-border-of-card-children"
+        <Col className="mt3 mb3" span={14}>
+          <Tabs
+            type="card"
+            tabBarStyle={{ marginBottom: 0 }}
```

Comment:
replacing it with ``mb0``?

Reply:
that's not possible unfortunately, it has to be a prob.